### PR TITLE
Infer variant type instead of passing it via cli

### PR DESCRIPTION
**Pull request by dcolligan**
Issue #186

### CODE REVIEW

File: ga4gh/datamodel/variants.py [hunk 1]
```diff
@@ -48,6 +48,13 @@ def convertVCFGenotype(vcfGenotype, vcfPhaseset):
     return genotype, phaseset
 
 
+def variantSetFactory(variantSetId, relativePath):
+    if True:  # TODO look for '.wt' in some part of directory path
```
Comment (dcolligan) [inline ga4gh/datamodel/variants.py]: @jeromekelleher can you provide some clarity here?  (see question on issue page)

Comment (jeromekelleher) [inline ga4gh/datamodel/variants.py]: Just use
```python
if variantSetId.endswith(".wt"): 
```
We're ripping out wormtable, so it doesn't need to be pretty.

**Comment by dcolligan** [comment]
@jeromekelleher pushed update

**Comment by jeromekelleher** [comment]
Nice, +1.

As an aside, the extra stuff you had to do here with the expected IDs makes me even more certain that hard coding strings in test cases is a bad idea...

**Comment by jeromekelleher** [comment]
Oops, I forgot - we need to update the README also, as this will now tell people the wrong thing. Can you add this here please @dcolligan?

**Comment by dcolligan** [comment]
@jeromekelleher pushed a new change with the README updated

**Pull request merged**